app/models/miq_expression.rb

```diff
@@ -1026,22 +1026,22 @@ def self.evaluate(expression, obj, _inputs = {}, tz = nil)
     result
   end
 
-  def evaluate(obj, inputs = {}, tz = nil)
-    self.class.evaluate(self, obj, inputs, tz)
+  def evaluate(obj, tz = nil)
+    self.class.evaluate(self, obj, tz)
   end
 
-  def self.evaluate_atoms(exp, obj, inputs = {})
+  def self.evaluate_atoms(exp, obj)
```

Comment:
This changed name in the first commit and was changed back in the second. Would it be possible to eliminate this change from the history?